code/game/objects/items/weapons/implants/implant_misc.dm

```diff
@@ -14,37 +14,41 @@
 	return dat
 
 
-/obj/item/weapon/implant/adrenalin
-	name = "adrenal implant"
-	desc = "Removes all stuns and knockdowns."
+/obj/item/weapon/implant/comstimms
+	name = "Combat Stimulant Implant"
+	desc = "A coctail of potent drugs will heal damage allowing you to fight for longer"
```

Comment:
A cocktail of potent drugs that will heal damage, allowing you to fight for longer.